src/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.svg'
```

Comment:
We don't need this once you pull in the latest master.

Reply:
> We don't need this once you pull in the latest master.

Alright, got it